telemetry/metrics.yaml

```diff
@@ -1,24 +1,270 @@
----
-# Schema
 $schema: moz://mozilla.org/schemas/glean/metrics/2-0-0
 
-page:
-  load:
+moso_events: # If we're using a single metrics.yaml across Mozilla Social, what do I call the category? Can I omit a category?
+
+  # First we have some event properties that are re-used across all events
+  # I implemented these as uuid/string/quantity metrics, instead of nesting them as extra_keys inside events. Is this valid?
+
+  # Is it possible to attach them to the baseline ping, as I've done?
+
+  mastodon_account_id:
+    type: string
+    lifetime: application
+    description: >
+      The user's account ID from Mastodon.
+    bugs:
+      - TBD
+    data_reviews:
+      - TBD
+    data_sensitivity:
+      - technical
+    notification_emails:
+      - kdemtchouk@mozilla.com
+    expires: never
+    send_in_pings:
+      - events
+      - glean_internal_info
+
+  mastodon_handle:
+    type: string
+    lifetime: application
+    description: >
+      The user's account handle.
+    bugs:
+      - TBD
+    data_reviews:
+      - TBD
+    data_sensitivity:
+      - technical
+    notification_emails:
+      - kdemtchouk@mozilla.com
+    expires: never
+    send_in_pings:
+      - events
+      - glean_internal_info
+
+  mastodon_instance_domain:
+    type: string
+    lifetime: application
+    description: >
+      The domain of the Mastodon instance where
+      the user's Mastodon account is hosted (e.g. `mozilla.social`).
+    bugs:
+      - TBD
+    data_reviews:
+      - TBD
+    data_sensitivity:
+      - technical
+    notification_emails:
+      - kdemtchouk@mozilla.com
+    expires: never
+    send_in_pings:
+      - events
+      - glean_internal_info
+
+  page_url:
```

Comment:
I don't think we need this as a top-level string metric if we're including it as an extra_key in the `page_view` event.

Reply:
From a client perspective this is easier as a top-level metric on web, if it is on all the events.